src/uct/api/uct.h

```diff
@@ -1339,6 +1341,29 @@ ucs_status_t uct_rkey_unpack(const void *rkey_buffer, uct_rkey_bundle_t *rkey_ob
 /**
  * @ingroup UCT_MD
  *
+ * @brief Get a pointer to a remote memory
+ *
+ * This routine returns a local pointer to the remote memory
+ * described by the rkey bundle. The MD must support
+ * @ref UCT_MD_FLAG_SHARE_PTR flag.
+ *
+ * @param [in]  rkey_ob      A remote key bundle as
+ *                           returned by the @ref uct_rkey_unpack
+ * @param [in]  remote_addr  A remote address within memory area described
+ *                           by the rkey_ob
+ * @param [out] addr_p       A pointer that can be used for the direct
```

Comment:
s/for the direct/for direct/

I also suggest, for consistency, to add a period at the end of the previous argument descriptions.